cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,31 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  const randomNumber = Math.random().toString().slice(2, 5);
+  const username = `test${randomNumber}`;
+  const email = `${username}@gmail.com`;
+  const password = 'Testpass1!';
+
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/');
```

Comment:
Avoid hardcoding the URL directly in the test. Instead, consider setting it in the cypress.json configuration file and use `cy.visit('/')` to visit the base URL.